addon/libconiks.js

```diff
@@ -0,0 +1 @@
+../chrome/content/libconiks.js
```

Comment:
This should link to `../chrome/content/coniks/libconiks.js` now.

But maybe we want to put this file in `addon/coniks/libconiks.js`, in which case it'll need an additional `../`

Annoyingly, the tests failed,
```
JPM [info] Starting jpm test on addon
JPM [error] Error: ENOENT: no such file or directory, stat '/home/travis/build/arlolra/ctypes-otr/addon/libconiks.js'
```
but didn't report the error :(